doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -473,13 +473,13 @@ Ltac2 models backtracking computations using streams of values: the head of a no
 
 Backtracking failures (empty streams) are moreover decorated with exceptions to inform the reason of failure. These exceptions are then passed to the tail of a non-empty backtracking computation. The following Ltac2 type summarizes the model for backtracking computations
 
-  .. rocqtop:: in
+.. rocqtop:: in
```

Comment:
`rocqtop` with `in` is the easiest use case (which could just as well be a `rocqdoc`).  The tricky part is when you're showing both input and output--did you try that use case?  For example, multiline comments that start after a command are tricky because of coqtop's behavior (see below).  I recall spending quite a bit of time on this.  It seemed like there were a lot of cases that weren't handled (including nested comments).

```
> coqtop

Coq < Print True. (* comment
Inductive True : Prop :=  I : True.

Coq < *) Print False.
Inductive False : Prop :=  .
```